benchmarks/dataframe.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+import string
+import time, argparse
+import numpy as np
+import pandas as pd
+
+import arkouda as ak
+import random
+
+OPS = ['_repr_html_', '_get_head_tail_server', '_get_head_tail']
+TYPES = ('int64', 'uint64',)
+
+def generate_dataframe(N):
+    types = [ak.Categorical, ak.pdarray, ak.Strings, ak.SegArray]
+
+    # generate random columns to build dataframe
+    df_dict = {}
+    for x in range(20):  # loop to create 20 random columns
+        key = f"c_{x}"
+        d = types[random.randint(0, len(types)-1)]
+        if d == ak.Categorical:
+            str_arr = ak.array(["".join(random.choices(string.ascii_letters + string.digits, k=5)) for _ in range(N)])
```

Comment:
This is fine for this benchmark's default problem size of 10**4, but just for future reference, the scalable way to generate a N-long ak.Strings array of 5-character strings is `ak.random_strings_uniform(minlen=5, maxlen=6, size=N)`.

Reply:
I did not know that we had this. I updated to use `ak.random_strings_uniform`.